src/Advection/vector_invariant_cross_upwinding.jl

```diff
@@ -0,0 +1,39 @@
+const VectorInvariantCrossVerticalUpwinding = VectorInvariant{<:Any, <:Any, <:Any, <:Any, <:AbstractUpwindBiasedAdvectionScheme, <:CrossAndSelfUpwinding}
+
+#####
+##### Cross upwinding results in the largest kinetic energy content,
+##### but because of presence of mixed upwinding leading to cross-double derivatives
```

Comment:
I think what you mean is, if you expand the truncation error, then the effective PDE being solved has cross-double derivatives. I think it's a good idea to explain this a little better